### PR TITLE
styled dashboard

### PR DESCRIPTION
**Pull request by tanishq-arya**
## Issue that this pull request solves

 Closes: #608

## Proposed changes

1. Added style for container
2. Box shadow to the cards in dashboard
3. Min max width for cards on dashboard
4. Center cards in dashboard on mobile view
 

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screenshot (1348)
Screenshot (1347)

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Admin/Components/Dashboard/dashboard.module.scss [hunk 1]
```diff
@@ -32,7 +33,7 @@
   font-size: 1.35rem;
   border-radius: 1em;
   height: auto;
-  width: 15em;
+  // width: 15em;
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/Dashboard/dashboard.module.scss:36]: please remove this if it's not required

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @tanishq-arya the current change is looking good.
can you please try reducing the left and right margin gap on the admin panel. Maybe shift the menu card a bit more left and increase the width of the card present on the right side?

**Comment by tanishq-arya** [comment]
> Hey @tanishq-arya the current change is looking good.
> can you please try reducing the left and right margin gap on the admin panel. Maybe shift the menu card a bit more left and increase the width of the card present on the right side?

On it

File: frontend/src/pages/Admin/Components/Dashboard/dashboard.module.scss (hunk 1 above)
Comment (tanishq-arya) [reply frontend/src/pages/Admin/Components/Dashboard/dashboard.module.scss:36]: ok

**Comment by tanishq-arya** [comment]
image

**Comment by Kajol-Kumari** [comment]
Hey @tanishq-arya thanks for incorporating the suggested changes quickly. Can we please have 3 cards in single row as there is a lot of left-out space between the cards on the right side. I believe it would look better that way.

**Comment by tanishq-arya** [comment]
> Hey @tanishq-arya thanks for incorporating the suggested changes quickly. Can we please have 3 cards in single row as there is a lot of left-out space between the cards on the right side. I believe it would look better that way.

I'll try that and update you soon.

**Comment by tanishq-arya** [comment]
image

**Comment by Kajol-Kumari** [comment]
Hey @tanay13 instead of reducing the card size, can we please make the use of the left and right side left-out spaces? let's make the navigation card to stick to left(with minimal required margin) and make the right card to cover the whole right side width(just leaving the minimal margin, same as the left side). It would look consistent that way.
Please let me know if something isn't clear to you.

**Comment by Kajol-Kumari** [comment]
Also, I can see that you have increased the width of the left-navigation card, which seems un-necessary to me.Can you please fix it?

**Comment by tanishq-arya** [comment]
image

You were talking about something like this right? And for the left it is responsive to 25% but I added a max-width:290px.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the broadcast and about-us cards that got distorted due to the changes you have made in this PR

**Comment by tanishq-arya** [comment]
> Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the `broadcast` and `about-us` and `faq` section cards that got distorted due to the changes you have made in this PR

Yes, sure. I was waiting for this to get approved.

**Comment by Kajol-Kumari** [comment]
> > Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the `broadcast` and `about-us` and `faq` section cards that got distorted due to the changes you have made in this PR
> 
> Yes, sure. I was waiting for dashboard view to get approved.

The dashboard changes are good to go. Please fix the other pages and this PR will be good to be merged :)

**Comment by tanishq-arya** [comment]
> > > Hey @tanishq-arya the dashboard is indeed looking good now. Can you please fix the `broadcast` and `about-us` and `faq` section cards that got distorted due to the changes you have made in this PR
> > 
> > 
> > Yes, sure. I was waiting for dashboard view to get approved.
> 
> The dashboard changes are good to go. Please fix the other pages and this PR will be good to be merged :)

Will push the changes soon.

**Comment by tanishq-arya** [comment]
image
@Kajol-Kumari Can you take a look at this and tell me if this size is okay?

**Comment by tanishq-arya** [comment]
Or should I make all the min-heights same ? This looks better to me as it brings a uniformity across the cards.
image

**Comment by tanishq-arya** [comment]
image
image
image
image

**Comment by tanishq-arya** [comment]
This is for the desktop view mobile view needs some changes.

**Comment by tanishq-arya** [comment]
image
Screenshot (1400)
Screenshot (1399)
image

**Comment by Kajol-Kumari** [comment]
Hey @tanishq-arya! For the sections having just two cards i.e `about-us` `broadcasts` and `faq` please keep the card dimension same as before as we will be having two cards in those pages and previous card size was looking better.

**Comment by tanishq-arya** [comment]
> Hey @tanishq-arya! For the sections having just two cards i.e `about-us` `broadcasts` and `faq` please keep the card dimension same as before as we will be having two cards in those pages and previous card size was looking better.

Okay, I'll make the changes.

**Comment by tanishq-arya** [comment]
@Kajol-Kumari I have reverted back on the changes and added changes only to the settings one. Please check this and let me know   if anything comes up. Also should I add any media queries for the 2 column ones - faq, broadcast, about in the mobile view ?

**Review by Kajol-Kumari — APPROVED** [review]
Hey @tanishq-arya thanks for incorporating the suggested changes.
LGTM 👍 

Can I please get a follow-up PR for 
1. Adding card effect to `broadcast, faq and about-us` similar to the cards present on the dashboard
2. Making them responsive for all screens
3. Reducing the width of the cards on these pages a bit (Just the width)

**Comment by tanishq-arya** [comment]
> Hey @tanishq-arya thanks for incorporating the suggested changes.
> LGTM 👍
> 
> Can I please get a follow-up PR for
> 
> 1. Adding card effect to `broadcast, faq and about-us` similar to the cards present on the dashboard
> 2. Making them responsive for all screens
> 3. Reducing the width of the cards on these pages a bit (Just the width)

Okay I ll raise a PR for that.

**Pull request merged**